### PR TITLE
chore: Update contributing docs

### PR DESCRIPTION
**Pull request by alimpens**
# Describe the pull request

## What

Updates to CONTRIBUTING.md

## Why

To make it clearer how users can open an issue and submit a PR

## How

By following an existing CONTRIBUTING.md template.

## Checklist

Before submitting your pull request, please ensure you have done the following. Check each checkmark if you have done so or if it wasn't necessary:

- [x] Add or update unit tests
- [x] Add or update documentation
- [x] Add or update stories
- [x] Add or update exports in index.\* files
- [x] Start the PR title with a Conventional Commit prefix, [as explained here](https://github.com/Amsterdam/design-system/blob/main/documentation/publishing.md?plain=1#L11).

## Additional notes

- Do we want to add a contributing section to Storybook as well?
- [I've created a ticket to add SUPPORT.md](https://gemeente-amsterdam.atlassian.net/browse/DES-886)
- Do we want to describe how we want to handle feature requests as well?

### CODE REVIEW

**Review by VincentSmedinga — COMMENTED** [review]
Great effort, lots of essential knowledge in here. I have some suggestions on specific sentences.

I am a bit worried about the usability of the document. On GitHub, [it looks like this](https://github.com/Amsterdam/design-system/blob/2e344a2a3d14655a32004a2c7eb92f5e584cec78/CONTRIBUTING.md) – the various lengths of sections, lists and underlines make it appear a bit messy. Although I don’t see an easy way out. I’d be fine with the current state as a good start.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,37 +1,106 @@
...
+
+Before creating an issue, check if you are using the latest version of the project. If you are not up-to-date, see if updating fixes your issue first.
+
+### Reporting security issues
+
+Review our [Security Policy](https://github.com/Amsterdam/.github/blob/main/SECURITY.md). **Do not** file a public issue for security vulnerabilities.
+
+### Bug reports and other issues
+
+For all other issues, you can [create an issue on GitHub](https://help.github.com/en/github/managing-your-work-on-github/creating-an-issue).
```
[the remark this answers is not in the document]
Comment (alimpens) [reply CONTRIBUTING.md:30]: Updated the link. I don't think the link to the GitHub docs is really necessary to be honest, it's pretty self-explanatory imo.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,37 +1,106 @@
...
+
+Follow these guidelines when doing so:
+
+- **Do not open a duplicate issue.** Search through existing issues to see if your issue has previously been reported. If your issue exists, comment with any additional information you have. You may simply note "I have this problem too", which helps prioritize the most common problems and requests.
```
[the remark this answers is not in the document]
Comment (alimpens) [reply CONTRIBUTING.md:34]: I agree. Shall I remove the last line?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,37 +1,106 @@
...
+
+- **Prefer using [reactions](https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)**, not comments, if you simply want to "+1" an existing issue.
+
+- **Fully complete the provided issue template.** The bug report template requests all the information we need to quickly and efficiently address your issue. Be clear, concise, and descriptive.
+
+## :repeat: Submitting pull requests
+
+Before opening a pull request for non-trivial changes, it is usually best to first open an issue to discuss the changes, or discuss your intended approach for solving the problem in the comments for an existing issue.
```
[the remark this answers is not in the document]
Comment (alimpens) [reply CONTRIBUTING.md:42]: Hm, I don't really think so. The original phrasing breaks it up in three parts, your change makes it harder to read imo.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,37 +1,106 @@
...
+
+- **Follow existing coding style and conventions.** Keep your code consistent with the style, formatting, and conventions in the rest of the code base. When possible, these will be enforced with a linter. Consistency makes it easier to review and modify in the future.
+
+- **[Follow our Definition of Done](https://github.com/Amsterdam/design-system/blob/main/documentation/definition-of-done.md)**.
+
+- **[Resolve any merge conflicts](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-on-github)** that occur.
+
+- **Promptly address any CI failures**. If your pull request fails to build or pass tests, please push another commit to fix it.
+
+### Local development
```
[the remark this answers is not in the document]
Comment (alimpens) [reply CONTRIBUTING.md:88]: The idea was that you need local development when submitting a PR, but I agree that it makes more sense to separate the sections. That also makes the document more readable probably.

[the remark this answers is not in the document]
Comment (VincentSmedinga) [reply CONTRIBUTING.md]: Yes, better.

File: CONTRIBUTING.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (VincentSmedinga) [reply CONTRIBUTING.md]: Yes please, I can post a suggestion.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,37 +1,133 @@
...
+### Bug reports and other issues
+
+For all other issues, you can [create an issue on GitHub](https://github.com/Amsterdam/design-system/issues).
+
+Follow these guidelines when doing so:
+
+- **Do not open a duplicate issue.**
+  Search through existing issues to see if your issue has previously been reported.
+  If your issue exists, comment with any additional information you have.
+  You may simply note "I have this problem too", which helps prioritize the most common problems and requests.
```
Comment (VincentSmedinga) [inline CONTRIBUTING.md:39]:
```suggestion
  Reacting with a ‘thumbs up’ helps us prioritize the most common problems and requests.
```

**Review by VincentSmedinga — APPROVED** [review]

**Pull request merged**